frank-doc-doclet/src/test/resources/doc/examplesExpected/simple.json

```diff
@@ -105,63 +117,128 @@
+      "elementNames":[
+        "RoleNameTChild"
+      ],
+      "attributes":[
+        {
+          "name":"active",
+          "description":"If defined and empty or false, then this element and all its children are ignored"
+        }
+      ]
     },
     {
-      "name": "Start",
-      "fullName": "org.frankframework.frankdoc.testtarget.examples.simple.Start",
+      "name":"Start",
+      "fullName":"org.frankframework.frankdoc.testtarget.examples.simple.Start",
       "description":"Description of Start. Value of VARIABLE is <code>variable value</code>. Value of TheEnum.ENUM_CONSTANT is <code>enum constant</code>.",
-      "parent": "org.frankframework.frankdoc.testtarget.examples.simple.AbstractParentOfStart",
-      "elementNames": ["Start"],
-      "attributes": [{
-        "name": "attribute"
-      }],
-      "children": [
+      "parent":"org.frankframework.frankdoc.testtarget.examples.simple.AbstractParentOfStart",
+      "elementNames":[
+        "Start"
+      ],
+      "attributes":[
+        {
+          "name":"attribute"
+        }
+      ],
+      "children":[
         {
           "multiple":true,
           "roleName":"module",
           "description":"Wrapper element to help split up large configuration files into smaller valid XML files. It may be used as root tag when an XML file contains multiple adapters and/or jobs. The Module element itself does not influence the behavior of Frank configurations.",
           "type":"Module"
         },
         {
-          "multiple": false,
-          "roleName": "roleNameIChild",
-          "type": "org.frankframework.frankdoc.testtarget.examples.simple.IChild"
+          "multiple":false,
+          "roleName":"roleNameIChild",
+          "type":"org.frankframework.frankdoc.testtarget.examples.simple.IChild"
         },
         {
-          "multiple": true,
-          "roleName": "roleNameTChild",
-          "type": "org.frankframework.frankdoc.testtarget.examples.simple.TChild"
+          "multiple":true,
+          "roleName":"roleNameTChild",
+          "type":"org.frankframework.frankdoc.testtarget.examples.simple.TChild"
         }
       ]
     }
   ],
-  "enums": [{
-    "name": "org.frankframework.frankdoc.testtarget.examples.simple.MyEnum",
-    "values": [
-      {
-        "label": "ONE"
-      },
-      {
-        "label": "customLabelTwo",
-        "description": "Description of TWO"
-      }
-    ]
-  }]
+  "enums":[
+    {
+      "name":"org.frankframework.frankdoc.testtarget.examples.simple.MyEnum",
+      "values":[
+        {
+          "label":"ONE"
+        },
+        {
+          "label":"customLabelTwo",
+          "description":"Description of TWO"
+        }
+      ]
+    }
+  ],
+  "properties":[
+    {
+      "name":"First group",
+      "properties":[
+        {
+          "name":"property.first",
+          "description":"This is a description",
+          "defaultValue":"Value",
+          "flags":[
+            [
+
+            ]
```

Comment:
Het is misschien wel mooier deze lijst leeg te laten wanneer er niets in staat. en ik zie dat je 2 arrays in elkaar hebt? 😄

Reply:
Woops dat is niet de bedoeling, gefixt!